rpc_jobs/multi_node_aio.yml

```diff
@@ -186,6 +207,10 @@
         }} finally {{
             pubCloudSlave.delPubCloudSlave(instance_name: instance_name)
             maas.entity_cleanup(instance_name: instance_name)
+            if(deploy_node != null){{
+              instance_name = deploy_node
+              ssh_slave.destroy()
```

Comment:
Also destroy should take a name param, so we don't have to set a global to different values between two calls of the function.